public/data/CBID2.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "Christine Belzie",
+	"type": "personal",
+	"bio": "An Instructional Designer who codes",
+	"avatar": "https://github.com/<CBID2>.png",
```

Comment:
The link is not working. I think it should be `https://github.com/CBID2.png` instead. So, remove the `< > `.

Reply:
Will do